pkg/uploader/types.go

```diff
@@ -43,7 +41,13 @@ type SnapshotInfo struct {
 	Size int64  `json:"Size"`
 }
 
+//UploaderProgress which defined two variables to record progress
```

Comment:
There is a same definition in https://github.com/vmware-tanzu/velero/blob/main/pkg/apis/velero/v1/pod_volume_operation_progress.go#L21, how about we use the existing one instead?

Reply:
this one `PodVolumeOperationProgress` is only used for statistic volume progress. 

`UploaderProgress` and `ProgressUpdater` are two generic definitions for progress modules, not only for volume progress.